src/test/java/armazem/ArmazemTest.java

```diff
@@ -0,0 +1,130 @@
+package armazem;
+
+import ingredientes.*;
+import armazem.Armazem;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.function.Executable;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ArmazemTest extends Armazem {
+
+    static Armazem armazem;
+    static Base baseSorvete;
+
+    @BeforeAll
+    void setupBase() {
+        baseSorvete = new Base(TipoBase.Sorvete);
+    }
+
+    @BeforeEach
+    void setup() {
+        armazem = new Armazem();
+        baseSorvete = new Base(TipoBase.Sorvete);
+    }
+
+    @Test
+    void testCadastrarIngredienteEmEstoque() {
+        armazem.cadastrarIngredienteEmEstoque(baseSorvete);
+        int quantidadeEmEstoque = armazem.consultarQuantidadeDoIngredienteEmEstoque(baseSorvete);
+
+        assertEquals(0, quantidadeEmEstoque);
+
+        IllegalArgumentException thrown = assertThrows(
+                IllegalArgumentException.class,
+                () -> armazem.cadastrarIngredienteEmEstoque(baseSorvete)
+        );
+        assertTrue(thrown.getMessage().contains("Ingrediente já cadastrado"));
+    }
```

Comment:
como boas práticas, sugere-se separar os testes "bons" dos testes "ruins"... outro ponto é a parte da nomenclatura, da uma olhada naquele artigo que coloquei lá no grupo. é interessante o teste dizer o que ele deve fazer e o que ele espera... veja o SHOULD... de resto ta massa demais!!!